chrome/index.html

```diff
@@ -130,28 +135,25 @@
                 </dl>
               </div>
               <svg id="mag">
-                <g class="grid x" transform="translate(40, 120)"></g>
+                <g class="grid x" transform="translate(40, 176)"></g>
                 <g class="grid y" transform="translate(40, 10)"></g>
                 <g class="data" transform="translate(41, 10)"></g>
-                <g class="axis x" transform="translate(40, 120)"></g>
+                <g class="axis x" transform="translate(40, 176)"></g>
                 <g class="axis y" transform="translate(40, 10)"></g>
               </svg>
             <div class="clear-both"></div>
           </div>
         </div>
-      <div id="input">
-        Input.
-      </div>
       </div>
     </div>
 
-    <div id="footer">
+    <!-- <div id="footer">
```

Comment:
I think it's a good idea to pop this up to the header div. Can you remove L150-156 here and remove the #footer CSS as well?